app/client/components/ListItemCheckbox.js

```diff
@@ -0,0 +1,42 @@
+
+import React from 'react';
+
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+
+export default class ListItemCheckbox extends React.Component {
+
+
+    static propTypes = {
+        isChecked: React.PropTypes.bool,
+        onChange: React.PropTypes.func,
+    }
+
+    static defaultProps = {
+        onChange: () => {},
+    }
+
+
+
+    render() {
+        var { isChecked } = this.props;
+        var { handleChange } = this;
+
+
+        var icon;
```

Comment:
this may be quite simple with the ternary operator:

```
var icon = isChecked ? 'check' : 'unchecked';
```

also the name of the variable it might be better as `iconName`?